server/.env.example

```
@@ -13,11 +13,26 @@ JWT_SECRET=
+# See .env.example at the repository root for a Compose-oriented combined template.
+
+DATABASE_URL=postgresql://postgres:Secret@2811@localhost:5432/internhack?sslmode=disable
+DODO_PAYMENTS_API_KEY=
+DODO_PAYMENTS_WEBHOOK_KEY=
+DODO_PAYMENTS_ENVIRONMENT=live_mode
+DODO_PRODUCT_ID_MONTHLY=
+DODO_PRODUCT_ID_YEARLY=
+ADZUNA_APP_ID=
+ADZUNA_APP_KEY=
+GEMINI_API_KEY=
+JWT_SECRET=0d571400b8dd9c0124f42046b1a8c6c9e5a4aec5bb09b51e3dff8247ffc24149
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The JWT_SECRET key is duplicated

(DuplicatedKey)